mongodb/springboot-multitenant/src/main/java/io/mongock/professional/examples/SpringBootMultitenantApp.java

```diff
@@ -45,18 +45,19 @@ private static SpringApplicationBuilder getSpringAppBuilder() {
     @Bean
     public RunnerSpringbootBuilder runnerSpringbootBuilderProfessional(ApplicationContext springContext, ApplicationEventPublisher eventPublisher) {
         // Runner
-        return MongockSpringboot.builder()
-                .setDriver(
-                        SpringDataMongoV3DriverProfessional.multiTenant()
-                                .addTenant(getDriver("test1"))
-                                .addTenant(getDriver("test2"))
-                                .addTenant(getDriver("test3"))
-                )
-                .addMigrationScanPackage("io.mongock.professional.examples.changelogs")
-                .setSpringContext(springContext)
-                .setEventPublisher(eventPublisher)
-                .setTrackIgnored(true)
-                .setTransactionEnabled(true);
+//        return MongockSpringboot.builder()
+//                .setDriver(
+//                        SpringDataMongoV3DriverProfessional.multiTenant()
+//                                .addTenant(getDriver("test1"))
+//                                .addTenant(getDriver("test2"))
+//                                .addTenant(getDriver("test3"))
+//                )
+//                .addMigrationScanPackage("io.mongock.professional.examples.changelogs")
+//                .setSpringContext(springContext)
+//                .setEventPublisher(eventPublisher)
+//                .setTrackIgnored(true)
+//                .setTransactionEnabled(true);
```

Comment:
@osantana85 this is now breaking. I don't if it's something I changed as I didn't spot because I didn't update the version inn this project or what? But could you take a look?